client/my-sites/plugins-wpcom/plugins-list.jsx

```diff
@@ -1,35 +0,0 @@
-/**
```

Comment:
We won't remove it but only for the feature flag. They should be updating a different file - `plugins-wpcom/index.jsx` or `plugins-wpcom/plugin-panel.jsx`